src/controllers/csi/provisioner/controller.go

```diff
@@ -156,28 +157,44 @@ func (provisioner *OneAgentProvisioner) updateAgentInstallation(ctx context.Cont
 	}
 	latestProcessModuleConfig = latestProcessModuleConfig.AddHostGroup(dk.HostGroup())
 
+	var agentUpdater *agentUpdater
 	if dk.CodeModulesImage() != "" {
+		// TODO: Maybe use the one in the Dynakube ?
```

Comment:
I'd prefer connection info from Dynakube because if it changes it meantime and is not reonciled yet, it makes us use two states (currently stored in dynakube and to-be-updated-with) at the time.